Sources/Sentry/SentrySysctl.m

```diff
@@ -35,7 +36,7 @@ + (void)load
     // there's no guarantee on whether that or this load method will be called first, the difference
     // in time has been observed to only be on the order of single milliseconds, not significant
     // enough to make a difference in outcomes
-    runtimeInitSystemTimestamp = getAbsoluteTime();
+    runtimeInitSystemTimestamp = [[[SentryDefaultCurrentDateProvider alloc] init] systemTime];
```

Comment:
We'll want a static access to `clock_gettime_nsec_np` since we want to do as little work as possible in the launch profile codepath. I see this as being a static method in SentryDefaultCurrentDateProvider that since there's really no underlying state anyways. This way, TestCurrentDateProvider could still override that for testing purposes (i haven't fully through this through, but it would need to provide access to the underlying state stored in the test mock subclass instance stored in the instance held by SentryDependencyContainer).